configs/components/rubygem-net-scp.rb

```diff
@@ -0,0 +1,17 @@
+component "rubygem-net-scp" do |pkg, settings, platform|
+  instance_eval File.read('configs/components/base-rubygem.rb')
```

Comment:
Since we're no longer building or shipping huaweios agents, we may be able to drop this and a other few rubygems from the runtime. 

Based on the set of huaweios-specific gems defined in the [puppet-agent project config](https://github.com/puppetlabs/puppet-agent/blob/e5503651654452d62b18a3da08c9ccd2539bd437/configs/projects/puppet-agent.rb#L275-L283), we may possibly be able to drop these from puppet-runtime:

- rubygem-net-ssh-telnet2
- rubygem-net-scp
- rubygem-mini_portile2
- rubygem-pkg-config
- rubygem-net-netconf
- rubygem-nokogiri

/cc @ScottGarman

Reply:
We should probably take this opportunity to remove references to huawei throughout the agent - I'll update this and make agent PRs unless anyone objects.